socorro/lib/revision_data.py

```diff
@@ -46,15 +46,17 @@ def get_revision_data():
     return {}
 
 
-def get_version():
+def get_version(revision_data=None):
     """Returns the Socorro version
 
     This pulls revision data and then returns the best version-y thing
     available: the tag, the commit, or "unknown" if there's no revision
     data.
 
+    :arg revision_data: revision data, or None to fetch from disk
     :returns: string
 
     """
-    revision_data = get_revision_data()
```

Comment:
I don't think the result of `get_version` will ever change during a run, so we could cache the result in the module and then return that. We do module-level caching in other places, too, so there's some precedent.

It's up to you.